[mds-daily] Add `undefined` handling to getVehicleCounts()

macsj200 opened the pull request:
## PR Checklist

 - [x] simple searchable title - `[mds-db] Add PG env var`, `[config] Fix eslint config`
 - [x] briefly describe the changes in this PR
 - [ ] mark as draft if should not be merged
 - [ ] write tests for all new functionality

## Impacts
- [ ] Provider
- [ ] Agency
- [ ] Audit
- [ ] Policy
- [ ] Compliance
- [x] Daily
- [ ] Native
- [ ] Policy Author

Commit:
Add `undefined` handling to getVehicleCounts()

Commit:
Add `undefined` handling to getVehicleCounts()

macsj200 committed:
[mds-compliance] Fix build (#203)

* [mds-compliance] fix type error

* remove unused import

jwtodd committed:
install: conditionally install tiller cluster role (#185)

* install: conditionally install tiller cluster role

* install/tiller: +uninstallTiller(), better helm-init, add warn(NOT FOR PRODUCTION USE)

avatarneil commented on packages/mds-daily/request-handlers.ts:
```diff
@@ -101,25 +109,29 @@ export async function getVehicleCounts(req: DailyApiRequest, res: DailyApiRespon
-              }
-              if (event.timestamp >= HRS_48_AGO) {
-                inc(stat.areas_48h, serviceArea.description)
+          if (item !== undefined) {
```
At the very least, we should log this case with a `WARN` or something to let us trace the root cause of this.

macsj200 replied:
Added, but I realized we don't really have any tests for either branch 🙀

avatarneil committed:
release/0.0.4 -> develop (#199)

* Publish

 - @aws-lambda/api-gateway-authorizer@0.1.10
 - @aws-lambda/aws-serverless-express-handler@0.1.10
 - @aws-lambda/aws-utils@0.1.10
 - @aws-lambda/mds-agency@0.1.10
 - @aws-lambda/mds-audit@0.1.11
 - @aws-lambda/mds-compliance@0.1.10
 - @aws-lambda/mds-daily@0.1.10
 - @aws-lambda/mds-metrics-sheet@0.1.11
 - @aws-lambda/mds-native@0.0.8
 - @aws-lambda/mds-policy-author@0.1.10
 - @aws-lambda/mds-policy@0.1.10
 - @aws-lambda/mds-provider@0.1.10
 - @container-images/env-inject@0.1.10
 - @container-images/mds-agency@0.1.10
 - @container-images/mds-audit@0.1.11
 - @container-images/mds-compliance@0.1.10
 - @container-images/mds-daily@0.1.10
 - @container-images/mds-metrics-sheet@0.1.11
 - @container-images/mds-native@0.0.8
 - @container-images/mds-policy-author@0.1.10
 - @container-images/mds-policy@0.1.10
 - @container-images/mds-provider@0.1.10
 - @mds-core/mds-agency@0.0.12
 - @mds-core/mds-api-authorizer@0.1.10
 - @mds-core/mds-api-helpers@0.1.10
 - @mds-core/mds-api-server@0.1.10
 - @mds-core/mds-audit@0.1.22
 - @mds-core/mds-cache@0.1.10
 - @mds-core/mds-compliance@0.1.11
 - @mds-core/mds-daily@0.0.11
 - @mds-core/mds-db@0.1.10
 - @mds-core/mds-metrics-sheet@0.0.11
 - @mds-core/mds-native@0.0.8
 - @mds-core/mds-policy-author@0.0.7
 - @mds-core/mds-policy@0.0.11
 - @mds-core/mds-provider@1.0.10
 - @mds-core/mds-providers@0.1.10
 - @mds-core/mds-stream@0.1.10
 - @mds-core/mds-test-data@0.1.10
 - @mds-core/mds-types@0.1.7
 - @mds-core/mds-utils@0.1.10

* Bump main package version

jwtodd committed:
helm values apis to keyed-map vs list for operational ease-of-use (#180)

* install: apis-as-maps

* helm/tests: green

* helm/test: remove test values files

* helm/tests: green

* helm/apis: support env blob

* helm: add mds service 'enabled' switch, true by default [ see: #MDSAAS-186 ]

* helm: provide ability to override default resources (ie cpu, memory)

* helm: camelCase override properties

* helm: use $key for service-id/name and remove redundant api.name

* helm: cleanup config setter

* helm: fallback from helm-upgrade trial

* install: mv bin/install bin/mdsctl

* helm: provide the ability to specify resource overrides scoped, in order of precedence, at the service-level, at the fleet (all services) and, lastly the provided devault values.

* README: update k8s readme

* helm: resouce override test; next: test overriding scoped to service

* helm/test: resource overrides scoped at api-service

janedotx committed:
[mds-db] Update geographies schema (#204)

* Initial schema changes.

* Add migration scaffolding (#192)

* Migration code for updating geos added.

Commit:
Install packages required by kubernetes install (#207)

jwtodd committed:
inline the mds-service image tags during the install:mds phase (#208)

* mdsctl: brew(oc,jq,gettext), installMds(infer versions)

* mdsctl: installMds infers image-version

* mdsctl: envsubst support

* mdsctl: purge images mds-* prior to build()

* mdsctl/image: more direct repo clean

jwtodd committed:
mdsctl: brew tap oq (#212)

Commit:
Merge branch 'fix/maxj/vehicle_counts' of github.com:CityOfLosAngeles/mds-core into fix/maxj/vehicle_counts

Commit:
add log statement for undefined case

Commit:
add type guard filter function

Commit:
fix bad test type

Commit:
add nifty filterEmpty function

avatarneil commented:
Please retarget this PR at @lacuna-tech/mds-core
